### ISSUE TITLE
vpshere cpi 54.x

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#662]
### Expected behavior
As a platform operator, i need up to date iaas provisionning components

### Observed behavior
as of v45 vsphere cpi is 52.x

### references
- https://github.com/cloudfoundry/bosh-vsphere-cpi-release/releases/tag/v53.0.4
- https://bosh.io/docs/vsphere-cpi/ 
- https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.resmgmt.doc/GUID-F8C7EF4D-D023-4F54-A2AB-8CF840C10939.html

**Comment by gberche-orange** [comment]
prereq of #612

**Comment by poblin-orange** [comment]
TODO: tests ok, must test micro bosh

**Comment by poblin-orange** [comment]
errors on brmc-int (bosh recreate)

```
Task 487179 | 10:30:48 | Preparing deployment: Preparing deployment (00:00:01)
Task 487179 | 10:30:49 | Preparing deployment: Rendering templates (00:00:01)
Task 487179 | 10:30:50 | Preparing package compilation: Finding packages to compile (00:00:00)
Task 487179 | 10:30:51 | Updating instance weave: weave/47426436-9e2a-4d72-8994-97808f77bd82 (0) (00:00:43)
                      L Error: Unknown CPI error 'Unknown' with message 'Content-Type is not supported: text/html' in 'detach_disk' CPI method (CPI request ID: 'cpi-492829')
Task 487179 | 10:31:34 | Error: Unknown CPI error 'Unknown' with message 'Content-Type is not supported: text/html' in 'detach_disk' CPI method (CPI request ID: 'cpi-492829')

```

**Comment by poblin-orange** [comment]
Analyse pb cpi vsphere 53 

    Une piste sérieuse vient de la gestion des tags qui a changée en 53+

    tags [Array, optional]: A list of category and tag name-value pairs used to attach to created VMs. Available in v53+. Available on vCenter 6.5+. The tags to be attached must be exist on vCenter host. For each tag to be attached, both category and tag names should be specified.

    Le vcenter est en version 6.5

    https://bosh.io/docs/vsphere-cpi/

    https://github.com/cloudfoundry/bosh-vsphere-cpi-release/blob/882f90114b67332c94e1ba83023ef3de9508f15b/src/vsphere_cpi/lib/cloud/vsphere/attach_tag_to_vm.rb#L94

    https://github.com/cloudfoundry/bosh-vsphere-cpi-release/blob/master/docs/required_vcenter_privileges.md

**Comment by ogrand** [comment]
Settings tags privileges on vsphere domain allow vm creation (proxy-r2 for testing) with v54 cpi version.

![tags_privileges](https://user-images.githubusercontent.com/26904966/84167526-a471b680-aa76-11ea-8226-c94ca83832a6.jpg)
